PSReadLine/Prediction.Views.cs

```diff
@@ -865,6 +917,146 @@ static StringBuilder AppendColor(StringBuilder buffer, string colorToUse, ref st
                 buffer.Insert(charPosition, " ", padding);
             }
 
+            /// <summary>
+            /// Generate the rendering text for the tooltip.
+            /// </summary>
+            private int RenderTooltip(string tooltip, List<StringBuilder> consoleBufferLines, ref int currentLogicalLine)
+            {
+                const int LengthOfLeadingPart = 6;
+                const string IndicatorSymbol = ">>";
+                const string MsgForViewAll = "(<F4> to view all)";
```

Comment:
Just curious if we do this elsewhere. I suspect that key binding is customizable by the user, but this won't update. Is that the case?

Reply:
Right, the <F4> key could already be bound by a user and this message currently won't update in that case.
We don't do this elsewhere -- first time advertising a key-binding in the rendering. Will collect the feedback and see how to best deal with it.

Reply:
Opened #3668 to track this.